_test/test_utilities/test_skip.m

```diff
@@ -0,0 +1,24 @@
+classdef test_skip < TestCase
+    properties
+    end
```

Comment:
You don't need to include a properties block if there are no properties to define.